prow/cmd/checkconfig/main.go

```diff
@@ -473,3 +532,21 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 	}
 	return errorutil.NewAggregate(queryErrors...)
 }
+
+func verifyOwnersPlugin(cfg *plugins.Configuration) error {
+	// we do not know the set of repos that use OWNERS, but we
+	// can get a reasonable proxy for this by looking at where
+	// the `approve', `blunderbuss' and `owners-label' plugins
+	// are enabled
+	approveConfig := enabledOrgReposForPlugin(cfg, approve.PluginName)
+	blunderbussConfig := enabledOrgReposForPlugin(cfg, blunderbuss.PluginName)
+	ownersLabelConfig := enabledOrgReposForPlugin(cfg, ownerslabel.PluginName)
+	ownersConfig := approveConfig.union(blunderbussConfig).union(ownersLabelConfig)
+	validateOwnersConfig := enabledOrgReposForPlugin(cfg, verifyowners.PluginName)
+
+	invalid := ownersConfig.difference(validateOwnersConfig).items()
+	if len(invalid) > 0 {
+		return fmt.Errorf("the following orgs or repos enable a plugin that uses OWNERS files but do not enable the %s plugin to ensure validity of OWNERS files: %v", verifyowners.PluginName, invalid)
```

Comment:
> enable a plugin that uses OWNERS files

We should list the plugins that use OWNERS files. If a user doesn't know all the plugins that use OWNERS files this error message may not be actionable.